packages/ios-client/Sources/FishjamClient/media/Tracks/LocalAppScreenShareTrack.swift

```diff
@@ -18,7 +18,7 @@ public class LocalAppScreenShareTrack: VideoTrack, LocalTrack {
         super.init(mediaTrack: mediaTrack, endpointId: endpointId, rtcEngineId: nil, metadata: metadata)
     }
 
-    convenience internal init(mediaTrack: RTCVideoTrack, oldTrack: LocalAppScreenShareTrack) {
+    convenience init(mediaTrack: RTCVideoTrack, oldTrack: LocalAppScreenShareTrack) {
```

Comment:
**chore:** Can you please also remove all other internals? 🙏

Reply:
Lets do it in seperate ticket when swift code would be refactored.